Aztec/Classes/TextKit/TextStorage.swift

```diff
@@ -231,6 +233,13 @@ open class TextStorage: NSTextStorage {
 
         return textStore.attributes(at: location, effectiveRange: range)
     }
+
```

Comment:
These changes (TextStorage) don't seem related to the crash fix.  Is that correct?